vector/v.colors/tests/test_v_colors.py

```diff
@@ -0,0 +1,284 @@
+import os
+import re
+import pytest
+import grass.script as gs
+import grass.exceptions
+
+
+@pytest.fixture
+def simple_vector_map(tmp_path):
+    """Fixture to create a simple vector map with 3 points and a numeric attribute column 'val'."""
+    project = tmp_path / "v_colors_project"
+    gs.create_project(project)
+
+    with gs.setup.init(project, env=os.environ.copy()) as session:
+        # Set the computational region
+        gs.run_command("g.region", n=10, s=0, e=10, w=0, res=1, env=session.env)
+
+        # Create a small ASCII file with coordinates
+        coords = "1|1\n2|2\n3|3"
+        path = tmp_path / "coords.csv"
+        path.write_text(coords)
+
+        # Import the points into a vector map
+        gs.run_command(
+            "v.in.ascii",
+            input=str(path),
+            output="test_points",
+            separator="|",
+            x=1,
+            y=2,
+            overwrite=True,
```

Comment:
Overwrite should not be needed since it should always start fresh each time the fixture is executed.